docs/modules/ROOT/pages/how-to/use-tilt.adoc

```diff
@@ -0,0 +1,179 @@
+= How to use Tilt with {product}
+
+image:logos/tilt.svg[role="related thumb right",alt="Tilt logo",width=120,height=120] https://tilt.dev/[Tilt] is a developer toolkit to enable teams to build microservices architectures faster on Kubernetes clusters.
+
+This page will describe the required steps to use Tilt successfully with {product}.
+
+== Requirements
+
+To follow this guide, please make sure that you have the following tools installed:
+
+`tilt`:: The https://docs.tilt.dev/[Tilt documentation] explains how to install it in your operating system of choice.
+
+`oc`:: You can download the OpenShift command directly from {product}, selecting the help menu (marked as a question mark) and selecting the "Command line tools" entry.
+
+`docker`:: You can download it from https://www.docker.com.
+
+== Procedure
+
+This section explains the steps required to use Tilt with {product}.
+
+=== 1. Login to {product}
+
+Follow these steps to login to {product} on your terminal and to create a project:
+
+. Login to the {product} console on your web browser.
+. Click on your user name on the top right and select "Copy login command"
+. Click "Display token" and copy the login command shown in "Log in with this token"
+. Paste the `oc login` command on the terminal:
++
+[source,shell]
+--
+$ oc login --token=sha256~_xxxxxx_xxxxxxxxxxxxxxxxxxxxxx-xxxxxxxxxx-X --server=https://api.${zone}.appuio.cloud:6443
+--
+
+. Create a new project called "fortune-go"
++
+[source,shell]
+--
+$ oc new-project fortune-go
```

Comment:
As with the language how-tos this will confusion if every user uses the same name.